feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/data/buyToken/MoonPayProvider.kt

```diff
@@ -16,7 +16,9 @@ class MoonPayProvider(
     private val publicKey: String,
 ) : ExternalProvider {
 
-    override val supportedTokens = emptySet<Chain.Asset>() // TODO wallet - buy
+    override val supportedTokens = mapOf(
+        "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3" to listOf("DOT")
```

Comment:
shouldn't this ID be received from JSON?

Reply:
should be - subject to discuss how to implement